packages/mdc-textfield/_mixins.scss

```diff
@@ -72,6 +74,12 @@
   }
 }
 
+@mixin mdc-text-field-input-hover-outline-stroke-color($color) {
```

Comment:
Ah this took me a while to figure out....I think you should add a comment about why this is not the same as `mdc-text-field-hover-outline-stroke-color`. Where does this get used?